app/pb_kits/playbook/pb_title_count/_title_count.scss

```diff
@@ -17,4 +17,13 @@
   &[class*=_right] {
     justify-content: flex-end;
   }
+
+  &.dark {
+    .pb_title_kit_3 {
+      @include pb_title_dark;
+    }
+    .pb_title_kit_4 {
+      @include pb_title_dark;
+    }
+  }
```

Comment:
Let's pass the dark prop to the child kits to handle the correct color rather than overrides